dso-l2/src/main/java/com/tc/services/PlatformServiceConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package com.tc.services;
+
+import org.terracotta.entity.ServiceConfiguration;
+import org.terracotta.monitoring.PlatformService;
+
+/**
+ * @author vmad
+ */
+public class PlatformServiceConfiguration implements ServiceConfiguration<PlatformService> {
```

Comment:
Are you able to use BasicServiceConfiguration for this (we should probably do a pass over our implementations since most just return a constant).